src/Symfony/Component/Form/AbstractTypeExtension.php

```diff
@@ -45,4 +45,13 @@ public function finishView(FormView $view, FormInterface $form, array $options)
     public function configureOptions(OptionsResolver $resolver)
     {
     }
+
+    public function getExtendedType()
+    {
+        @trigger_error(sprintf('The %s::getExtendedType() method is deprecated since version 4.1 and will be removed in Symfony 5.0. Use the %s::extendsType() method instead.', get_class($this), get_class($this)), E_USER_DEPRECATED);
+
+        if (method_exists($this, 'extendsType')) {
+            return $this::extendsType();
+        }
```

Comment:
you must throw an exception here, telling to implement the other method or to override this one, because returning `null` is not allowed.